acbuild/acbuild.go

```diff
@@ -239,3 +262,30 @@ func stdout(format string, a ...interface{}) {
 	out := fmt.Sprintf(format, a...)
 	fmt.Fprintln(os.Stdout, strings.TrimSuffix(out, "\n"))
 }
+
+func addACBuildAnnotation(a *lib.ACBuild, cmd *cobra.Command, args []string) error {
+	if a == nil {
```

Comment:
It gets called with nil here: https://github.com/appc/acbuild/pull/146/files#diff-003fdde72cd1c20a1cfa75450f56e198R128